Zend/zend_vm_execute.h

```diff
@@ -50809,6 +50812,14 @@ static ZEND_OPCODE_HANDLER_RET ZEND_FASTCALL ZEND_NULL_HANDLER(ZEND_OPCODE_HANDL
 
 ZEND_API void execute_ex(zend_execute_data *ex)
 {
+#if (ZEND_VM_KIND == ZEND_VM_KIND_HYBRID)
+	if (EXPECTED(execute_data != NULL)) {
+		ZEND_INCR_VM_COUNT();
```

Comment:
Maintain the `vm_count` here seems not a good idea, as many extension will use their own `execute_ex`.

What about put them before/after all the `internal_function.handler(...)` @dstogov ?